REA/Views/UserManagementPage.xaml

```diff
@@ -2,22 +2,50 @@
 <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              x:Class="REA.Views.UserManagementPage"
+             xmlns:vm="clr-namespace:REA.ViewModels"
              Title="User Management">
+    <ContentPage.BindingContext>
+        <vm:UserManagementViewModel />
+    </ContentPage.BindingContext>
 
-    <VerticalStackLayout Padding="20">
-        <Label Text="Select a User:" FontSize="20"/>
-
-        <ListView x:Name="UsersListView" SelectionMode="Single" ItemSelected="OnUserSelected">
+    <VerticalStackLayout Padding="20" Spacing="20">
+        <Label Text="Select a User:" FontSize="20" />
+        
+        <!--All users (displaying username and user id)-->
+        <ListView x:Name="UsersListView"
```

Comment:
Something to fix if there's time later is the bounds of this list view, when data is loaded after the layout MAUI seems to bug out with the height leading to
![Screenshot 2025-04-13 at 16 15 47](https://github.com/user-attachments/assets/dda43380-fb8b-42d2-ac97-cfcdfccd7867)

I was able to fix it with the following

```diff
        <ListView x:Name="AlertListView"
                  SelectionMode="Single"
                  ItemSelected="OnAlertSelected"
                  ItemsSource="{Binding Alerts}"
+                 MinimumHeightRequest="400"
                  >
```

But there may be another way to fix it by forcing the content page to rerender.